src/promptflow-tools/requirements.txt

```diff
@@ -1,4 +1,3 @@
 google-search-results==2.4.1
-promptflow
```

Comment:
MARK: wait promptflow tools team to delete this

Reply:
Yes, pipelines also need to adapt.

Reply:
let me remove this.